go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/ikeikeikeike/go-sitemap-generator/v2
+
+go 1.9
```

Comment:
I just picked this since tests seemed to work fine on `1.9.7` and anything earlier than 1.9 won't read this file anyways.